examples/src/routes.tsx

```diff
@@ -69,15 +71,17 @@ export const exampleRoutes: Array<ExampleRoute> = [
     name: 'default',
     path:
       `/migration?project=${project}` +
+      `&env=${defaultEnvironmentParam}` +
       `&modelId=${cadId}` +
       `&revisionId=${cadRevisionId}`,
-    menuTitle: 'Default',
+    menuTitle: 'Abandon all hope ye who enter here',
```

Comment:
wat

Reply:
Damnit, hoped you wouldn't notice :sweat_smile: